src/components/modal/personalization/scroll/personalization-scroll.modal.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, TypographyScale, Checkbox } from '@mdb/flora';
+import { TopicCard } from '@mdb/devcenter-components';
+import { useModalContext } from '../../../../contexts/modal';
+import tagConfig from '../../../../service/get-personalization-modal-config.preval';
+import { Tag } from '../../../../interfaces/tag';
+import { ScrollModalProps } from '../types';
+import { submitPersonalizationSelections } from '../utils';
+import { tagToTopic } from '../../../../utils/tag-to-topic';
+
+import styles from '../styles';
+
+const ScrollPersonalizationModal = ({
+    title,
+    subtitle = '',
+    existingSelections = [],
+}: ScrollModalProps) => {
+    const { closeModal } = useModalContext();
+    const [isOptedIn, setIsOptedIn] = useState(true);
+    const [selections, setSelections] =
+        useState<Array<Tag>>(existingSelections);
+
+    const onCheckboxToggle = (isChecked: boolean) => setIsOptedIn(isChecked);
+
+    const onSelectionToggle = (tag: Tag, wasSelected: boolean) => {
+        if (wasSelected) {
+            return setSelections(currSelections =>
+                currSelections.filter(sel => sel.name !== tag.name)
+            );
+        }
+
+        return setSelections(currSelections => [...currSelections, tag]);
+    };
+
+    const onCompletion = () => {
+        submitPersonalizationSelections({
+            preferences: selections,
+            emailPreference: isOptedIn,
+        });
+        closeModal();
+    };
+
+    return (
+        <div sx={styles.wrapper}>
+            <TypographyScale
+                variant="heading5"
+                sx={{ ...(!subtitle && styles.scrollHeader) }}
+            >
+                {title}
+            </TypographyScale>
+            {subtitle && (
+                <TypographyScale variant="body2" sx={styles.subtitle}>
+                    {subtitle}
+                </TypographyScale>
+            )}
+            <div sx={styles.tagsWrapper}>
+                {tagConfig.map(({ title, tags }) => (
+                    <Fragment key={title}>
+                        <TypographyScale
+                            variant="heading6"
+                            sx={styles.categoryHeader}
+                        >
+                            {title}
+                        </TypographyScale>
+                        <Grid
+                            columns={[2, null, null, 3]}
+                            sx={styles.scrollTagSection}
+                        >
+                            {tags.map(tag => {
+                                const isSelected = !!selections.find(
+                                    prevTags => prevTags.name === tag.name
+                                );
+                                // do not pass slug b/c it would create a hyperlink on the card
+                                const { icon, title } = tagToTopic({
+                                    ...tag,
+                                    slug: '',
+                                });
```

Comment:
This is my bad. I should have just removed the `href` prop from the `ThirdPartyLogo` [here](https://github.com/mongodb/devcenter/blob/main/src/utils/tag-to-topic.tsx#L25). There's no reason for the icon to be an anchor. If you remove the `href` prop from being passed in that file, you should be able to just pass the whole tag and not have to worry about an anchor being rendered.

Reply:
@mcolella14 i tried that out and it seems like the lighthouse tests were failing removing the `href` prop from `<ThirdPartyLogo />`. Not sure why that would be the case, reverted for now but will try to find out whats causing that

Reply:
Ah damn that's annoying, looks like flora still renders them as anchors even though they have no href https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1675196997595-248.report.html#crawlable-anchors

Reply:
@mcolella14 lol figured it was going to be something like that, ill look into their component and see if just conditionally setting the prop will alleviate it

Reply:
ahh ok, that's fine with me